build/cmd/zonedb/main.go

```diff
@@ -176,7 +180,19 @@ func main() {
 		}
 	}
 
-	if *updateIDNTables || *updateAll {
+	// Explicitly _not_ under updateAll at this time, until we're confident this
+	// won't get us blacklisted on IANA's web-servers
+	if *updateIDNURLs {
```

Comment:
Since this only fetches a single HTML file, shouldn’t this be safe under `*updateAll`?